src/index.html

```diff
@@ -10,5 +10,72 @@
   </head>
   <body>
     <h1>Stop watch</h1>
+    <div class="stopwatch">
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
+      <div class="stopwatch__scale">|</div>
```

Comment:
Code looks too complicated but nice styling 🔥

Reply:
Thank you so much. I know. But I couldn`t come up with anything else.